src/components/panel/panel.js

```diff
@@ -895,7 +898,12 @@ MdPanelService.prototype.create = function(config) {
   this._config.scope.$on('$destroy', angular.bind(panelRef, panelRef.detach));
 
   if (this._config.groupName) {
-    panelRef.addToGroup(this._config.groupName);
+    if (angular.isString(this._config.groupName)) {
+      this._config.groupName = [this._config.groupName];
```

Comment:
The `groupName` here refers to the parameter that the user passes from the panel configuration. Since it can be either a single groupName or an array of groupNames, I was concerned about changing its name.

@ErinCoughlan what do you think about the naming convention here? Let's get the whole team's buy-in.